Modules/_abc.c

```diff
@@ -464,7 +676,8 @@ _abc_subclasscheck(PyObject *self, PyObject *args)
     if (incache < 0) {
         return NULL;
     }
-    if (_get_negative_cache_version(self) < abc_invalidation_counter) {
+    if (PyObject_RichCompareBool(_get_negative_cache_version(self),
```

Comment:
_get_negative_cache_version could return null

Reply:
also, PyObjectRichCompareBool could have an error and return -1, which is silently treated as false